meta-mender-tegra/recipes-bsp/tegra-binaries/redundant-boot-overrides/update-nvbootctrl.sh.in

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+NVBC=@SBINDIR@/nvbootctrl
+curslot=$($NVBC get-current-slot)
+if [ -n "$curslot" ]; then
+    if $NVBC set-active-boot-slot $curslot && $NVBC mark-boot-successful; then
```

Comment:
@madisongh what do you think about adding a sentinel file at `/var/lib/mender/dont-mark-next-boot-successful` or similar which turns off this logic so we could test rollback with automation in cases where we don't support `mender rollback`?

Reply:
I could add a
```
ConditionPathExists=!/var/lib/mender/dont-mark-next-boot-successful
```
line to the service file, so it won't get started at all if the sentinel exists.  How does that sound?

Reply:
> I could add a
> ConditionPathExists=!/var/lib/mender/dont-mark-next-boot-successful
> line to the service file, so it won't get started at all if the sentinel exists. How does that sound?

Good idea.

> Either way, you'll have to remember to remove that sentinel file to restore normal operation.

Correct, our thought is to have some automation that prevents this step from happening, then do enough reboots to prove the bootloader switches to rollback partition, then remove the file to go back to normal operation.

Reply:
OK, I've added that, rebased the changes and added one more commit to fix a minor build warning that's bugging me for a while.